modular_skyrat/modules/moretraitoritems/code/traitor_announcer.dm

```diff
@@ -0,0 +1,50 @@
+/obj/item/device/traitor_announcer
+	name = "odd device"
+	desc = "Hmm... what is this for?"
+	special_desc_requirement = EXAMINE_CHECK_SYNDICATE
+	special_desc = "A remote usable to fake an announcement from the captain, of your choice."
+	icon = 'icons/obj/device.dmi'
+	lefthand_file = 'icons/mob/inhands/items/devices_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/items/devices_righthand.dmi'
+	icon_state = "inspector"
+	worn_icon_state = "salestagger"
+	inhand_icon_state = "electronic"
+	///How many uses does it have? -1 for infinite
+	var/uses = 1
+
+/obj/item/device/traitor_announcer/attack_self(mob/living/user, modifiers)
+	. = ..()
+	//can we use this?
+	if(!isliving(user) || !uses)
+		balloon_alert(user, "no charge!")
+		return
+	//build our announcement
+	var/origin = sanitize_text(reject_bad_text(tgui_input_text(user, "Who is announcing, or where is the announcement coming from?", "Announcement Origin", get_area_name(user), max_length = 28)))
+	var/audio_key = tgui_input_list(user, "Which announcement audio key should play? ('Intercept' is default)", "Announcement Audio", GLOB.announcer_keys, ANNOUNCER_INTERCEPT)
```

Comment:
The list here should be curated or just use a single default sound; there's a lot of unbalanced audio files in this (especially our modular ones...) and lots of very specific announcement tracks (like, specifically saying that something happened, i.e. grav gen offline). 
Not much to gain from letting players choose those when very few will end up fitting announcements they make anyways. Hell, most people just have announcer sounds turned off anyways...